### PR TITLE
translate supported-doc-versions

### PR DESCRIPTION
**Pull request by xiexiaojing**
translate supported-doc-versions into Chinese.

### CODE REVIEW

**Comment by xiexiaojing** [comment]
/assign @SataQiu  @xiangpengzhao 
I resubmit the PR. Would you please review the code for me? Thank you so much!

File: content/zh/docs/home/supported-doc-versions.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+本网站包含了当前最新版本和之前四个版本的 Kubernetes.
+
+{{% /capture %}}
+
+{{% capture body %}}
+
+## 当前最新版本
+
+The current version is
```
Comment (SataQiu) [inline content/zh/docs/home/supported-doc-versions.md]: 这句没有翻译

File: content/zh/docs/home/supported-doc-versions.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+content_template: templates/concept
+card:
+  name: about
+  weight: 10
+  title: Kubernetes 文档支持的版本
+---
+
+{{% capture overview %}}
+
+本网站包含了当前最新版本和之前四个版本的 Kubernetes.
```
Comment (SataQiu) [inline content/zh/docs/home/supported-doc-versions.md]:
```suggestion
本网站包含了当前最新版本和之前四个版本的 Kubernetes。
```

File: content/zh/docs/home/supported-doc-versions.md [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+
+{{% /capture %}}
+
+{{% capture body %}}
+
+## 当前最新版本
+
+The current version is
+[{{< param "version" >}}](/).
```
Comment (SataQiu) [inline content/zh/docs/home/supported-doc-versions.md]:
```suggestion
[{{< param "version" >}}](/)。
```

File: content/zh/docs/home/supported-doc-versions.md (hunk 2 above)
Comment (xiangpengzhao) [inline content/zh/docs/home/supported-doc-versions.md]: 英文句点改为中文句号

File: content/zh/docs/home/supported-doc-versions.md (hunk 1 above)
Comment (xiangpengzhao) [inline content/zh/docs/home/supported-doc-versions.md]: 这句话需要翻译出来 :)

File: content/zh/docs/home/supported-doc-versions.md (hunk 3 above)
Comment (xiexiaojing) [reply content/zh/docs/home/supported-doc-versions.md]: Thank you!

**Comment by xiangpengzhao** [comment]
/approve
Thanks @xiexiaojing !

**Pull request merged**